src/components/SecretsModal/BasicAuthFields.js

```diff
@@ -26,15 +26,13 @@ const BasicAuthFields = props => {
     serviceAccounts
   } = props;
 
-  const saItems = serviceAccounts
-    .filter(sa => sa.metadata.namespace === namespace)
-    .map(sa => (
-      <SelectItem
-        value={sa.metadata.name}
-        text={sa.metadata.name}
-        key={`${sa.metadata.namespace}:${sa.metadata.name}`}
-      />
-    ));
+  const saItems = serviceAccounts.map(sa => (
```

Comment:
since the `namespace` prop is no longer being used it should be removed from line 22 and also from any tests / other code that are currently passing it.

Reply:
the namespace prop is used in `line 69` to see if the Service Account Dropdown should be disabled or not.

Reply:
ah so it is 👍